commons/src/main/java/com/powsybl/commons/io/table/TableFormatter.java

```diff
@@ -30,4 +34,10 @@
     TableFormatter writeCell(boolean b) throws IOException;
 
     @Override void close() throws IOException;
+
+    TableFormatter writeCell(int i, CellStyle cellStyle) throws IOException;
+
+    TableFormatter writeCell(float f, CellStyle cellStyle, String stringFormat) throws IOException;
+
+    TableFormatter writeCell(double d, CellStyle cellStyle, String stringFormat) throws IOException;
```

Comment:
Move these 3 methods near the others. You should probably add writeCell methods with only the value as parameters